src/pages/ResetPassword.js

```diff
@@ -0,0 +1,21 @@
+import { Link } from "react-router-dom";
+
+export function ResetPassword(){
+    return <div className="login-register">
+        <h1 className="login-register-heading">Reset Password</h1>
+        <form>
+            <div>
+                <label >
+                    Email address
+                    <input placeholder="Enter Email" type="email" required></input>
+                </label>
+            </div>
+            <button className="primary-btn">
+                SUBMIT
```

Comment:
Reset password functionality doesn't work.

Reply:
I am working on it. Will resolve it soon.